GameOfRunes/GameOfRunes/Models/Stage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Stage.swift
+//  GameOfRunes
+//
+//  Created by Jermy on 29/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import UIKit
+
+struct Stage {
+    enum Category: String {
```

Comment:
do we need a category? Shouldn't every stage have a boss?

Reply:
My idea is sth like a "boss" stage to mark the end of a "Chapter". Each Chapter represents a series of stages in a region of the map.